.github/workflows/package.yaml

```diff
@@ -0,0 +1,55 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
```

Comment:
@daavoo The registry is not needed here, by default it does the login process in DockerHub. You only have to include it in the image tag, is not necessary in the login: 

https://github.com/mozilla-ai/lumigator/blob/main/.github/workflows/build_images_uv.yaml#L47
https://github.com/mozilla-ai/lumigator/blob/main/.github/workflows/build_images_uv.yaml#L62